src/utils.ts

```diff
@@ -54,47 +55,49 @@ export function readBigUInt64LE(buffer: Buffer, offset = 0) {
 export function parseDipState(buf: Buffer) {
   const strike = Number(readBigUInt64LE(buf, 8));
   const expiration = Number(readBigUInt64LE(buf, 16));
-  const splMint = new PublicKey(buf.slice(24, 56));
+  const baseMint = new PublicKey(buf.slice(24, 56));
   const vaultMint = new PublicKey(buf.slice(56, 88));
   const vaultMintBump = Number(buf.readUInt8(88));
   const vaultSpl = new PublicKey(buf.slice(89, 121));
   const vaultSplBump = Number(buf.readUInt8(121));
   const optionMint = new PublicKey(buf.slice(122, 154));
   const optionBump = Number(buf.readUInt8(154));
-  const vaultUsdc = new PublicKey(buf.slice(155, 187));
-  const vaultUsdcBump = Number(buf.readUInt8(187));
-  const usdcMint = new PublicKey(buf.slice(188, 220));
+  const vaultPremium = new PublicKey(buf.slice(155, 187));
+  const vaultPremiumBump = Number(buf.readUInt8(187));
+  const quoteMint = new PublicKey(buf.slice(188, 220));
```

Comment:
Not sure if some of these are redundant?

Reply:
quoteMint and quoteAsset? They are reading the same thing, so should only need one.

Reply:
vaultPremium & quoteMint/ quoteAsset

Reply:
Most of the stuff in here isnt actually needed, was just included for completeness of parsing the program account.